lib/features/local_live_log.js

```diff
@@ -164,6 +164,14 @@ export default class TaskclusterLogs {
 
   async killed(task) {
     debug('switching live log redirect to backing log...')
+    try {
+      await this.container.stop();
```

Comment:
And what did we do before?
Did we just leave it for the GC... so that it could finish serving existing connections?